src/lib/timeframeConstants.js

```diff
@@ -0,0 +1,6 @@
+const SOON = {stringValue: "fewer that 7 days", integerValue: 7}
+const KIND_OF_SOON = {stringValue: "between 7 and 30 days", integerValue: 14}
+const NOT_SOON = {stringValue: "more than 30 days", integerValue: 30}
+const INACTIVE = {stringValue: "inactive", integerValue: 0}
+
```

Comment:
Brings a full circle feel to an earlier issue for Diane and I, thank you!

Reply:
Awesome. Great idea.